jumanji/environments/combinatorial/binpack/env_viewer.py

```diff
@@ -24,23 +26,38 @@
 from jumanji.environments.combinatorial.binpack.types import State, item_from_space
 
 
+class RenderMode(str, Enum):
```

Comment:
Why are you inheriting from `str`? Why not like [this](https://docs.python.org/3.8/library/enum.html#creating-an-enum)?

Reply:
If it inherits from str as well, it can be instantiated from a str parameter, like `render_mode = RenderMode("human")`. As such we can accept either a str or an enum instance in the constructor of the env. Otherwise turning a string to an instance is quite painful

It is kind of like [StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum) before python 3.11